lms/djangoapps/certificates/apis/v0/permissions.py

```diff
@@ -0,0 +1,30 @@
+"""
+This module provides a custom DRF Permission class for supporting the course certificates
+to Admin users and users whom they belongs to.
+"""
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from rest_framework.permissions import IsAdminUser
+
+from openedx.core.djangoapps.user_api.accounts.serializers import get_profile_visibility
+
+
+class IsOwnerOrPublicProfile(IsAdminUser):
```

Comment:
If you leave this as-is, I'd rename to `IsStaffOrOwnerOrPublicCertificates`.
* Note: I added "IsStaff" and I renamed "PublicProfile" to "PublicCertificates". Or, you could use "PublicProfileCertificates".

Nits:
* Also, alternatively, I think you could remove `IsAdminUser` from here and in `permission_classes` below you could use `IsAdminUser|IsOwnerOrPublicProfile`.
* Additionally, if you prefer `IsStaff` to `IsAdminUser`, I think it does the same thing with a different name. See https://github.com/edx/edx-drf-extensions/blob/master/edx_rest_framework_extensions/permissions.py#L26